internal/appsec/remoteconfig.go

```diff
@@ -70,29 +70,49 @@ updateLoop:
 			statuses = mergeMaps(statuses, status)
 			r.AddEdit("asmdata", config.RulesFragment{RulesData: rulesData})
 		case rc.ProductASMDD:
-			// Switch the base rules of the RulesManager if the config received through ASM_DD is valid
-			// If the config was removed, switch back to the static recommended rules
-			if len(u) > 1 { // Don't process configs if more than one is received for ASM_DD
-				log.Debug("appsec: Remote config: more than one config received for ASM_DD. Updates won't be applied")
-				err = errors.New("more than one config received for ASM_DD")
-				statuses = mergeMaps(statuses, statusesFromUpdate(u, true, err))
-				break updateLoop
-			}
+			removalFound := false
+			var newBasePath string
+			var newBaseData []byte
```

Comment:
(Very optional, such nitpick) Would rather consistent style this:

```suggestion
			var (
				removalFound = false
				newBasePath string
				newBaseData []byte
			)
```